thevetdoctor/teamui
# #29823200:create a page/feature to edit articles

What does this PR do?
create a page/feature to edit articles

Description of Task to be completed?
create a page/feature to edit articles

How should this be manually tested?
Navigate to /feed
Click the edit button on a specific article, supply the new title and text of article, and click update
or click cancel if edit is no longer required

Any background context you want to provide? NA
What are the relevant pivotal tracker stories?
Github Project Board : #-card-29823200

Screenshots (if appropriate):NA

Questions: NA
